src/tests/unit/tests/DetailsView/components/failure-instance-panel-control.test.tsx

```diff
@@ -148,10 +154,13 @@ describe('FailureInstancePanelControlTest', () => {
             .props()
             .onDismiss();
 
+        clearPathSnippetDataMock.setup(handler => handler()).verifiable(Times.once());
```

Comment:
These tests might read more naturally if the setup were done before the action to verify - e.g. calling `dismiss` presumably invokes `clearPathSnippetDataMock` but the only lines between setup/verify here are `expects`. I'm somewhat surprised the `Times.once` verification passes

Reply:
Gotcha yes--I think I was thinking about it as if I were putting the clearPathSnippetDataMock where it would be called in the real order, but I realize now that the *set up* of it makes more sense before, and that's what we're really doing in that line (setting up, not calling the function)